app/services/validateCredentials.js

```diff
@@ -37,7 +37,7 @@ module.exports.signIn = async body => {
   const expireTime = moment(now)
     .add(TOKEN_EXPIRATION, 'seconds')
     .format('MMMM Do YYYY, h:mm:ss a');
-  const tokenArray = { token: newSession.id, email: user.email, expireTime };
+  const tokenArray = { session_id: newSession.id, email: user.email, expireTime };
```

Comment:
Uhg, my bad on this one, I brainfarted and asked for `session_id` in `snake_case` when `expireTime` was in `camelCase`. I'm pretty sure we don't have an standard for tokens, but its params should at least be consistent.